packages/strapi-helper-plugin/lib/src/components/AutoReloadBlocker/index.js

```diff
@@ -0,0 +1,48 @@
+/**
+*
+* AutoReloadBlocker
+*
+*/
+
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import styles from './styles.scss';
+
+function AutoReloadBlocker() {
+  return (
+    <div className={styles.autoReloadBlocker}>
+      <div className={styles.header}>
+        <div>
+
+          <h4>
+            <FormattedMessage id="components.AutoReloadBlocker.header" />
```

Comment:
Wouldn't better to create a single, but dynamic Blocker component, that support props like `title`, `description` and `content` (which could even be a component defined in the plugin)?

Reply:
The plugin is in charge of telling which component to render if we prevent it from being displayed. In this case passing such logic is a bit more complex and since we're rendering stateless component  instead of the plugin itself I thought that having two differents blocker is a great deal.